about.html

```diff
@@ -0,0 +1,117 @@
+     <!-- <h3 class="h3 text-white-50 container text-center pt-5 pb-1">"Connect with like minds"</h3> -->
+     <div class="head container py-4">
+       <h1 class="h1 headtext pt-5 text-center">About Women In Tech General Summit</h1>
+       <div class="sep mx-3 my-3"></div>
+
+      </div>
+     <p class="theme h4 px-2 text-center">Unite 100 000 Women in Tech to Foster Innovation with Purpose and Impact</p>
+
+     <div class="about-us container px-4 py-2">
+        <p class="text-white-50">In the past few years there has been a flourishing number of self-organisation amongst African women in technology. Many focus on encouraging women entering the field as coders, IT managers and to establish enterprises that increase women’s socio-economic well-being. Projects have also focused on improving self-reliance and livelihoods in rural areas.
+
+            In order to identify and learn from other Women in Technology organisations in Africa, Women in Technology Ghana spear headed and run the first Pan African women in tech virtual meet up which brought together over 150 women in technology roles across Africa and was featured in CNN.
+            
+            The success of this process resulted established a formal Pan-African Network of Women in Technology to organise regular activities to provide effective support to women in technology across the African continent.</p>
+     </div>
+    </main>
+  </section>
+  <section class="mission py-5">
+    <div class="program-box d-flex flex-column justify-content-center align-items-center">
+        <h2 class="h2 text-center pt-5">Our Mission</h2>
+        <div class="pt-1 underprogram"></div>
+      </div>
+    
+
+    <div class="wo-mission container py-4 px-4">
+      <h5 class=" mission-text py-2 px-2 text-center">
+        • Creating today’s female leaders and role Models for tomorrow’s Women<br>
+        • Showing the world what a strong woman in Tech capable of achieving<br>
+        • Support World growth through technology    
+      </h5>
+    </div>
+  </section>
+<!-- END MISSION -->
+<!-- SEE PAST SUMMIT -->
+  <section class="see container-fluid">
+    <div class="seepast px-4 py-5">
+      <div class="program-box d-flex flex-column justify-content-center align-items-center">
+        <h2 class="h2 text-center pt-5">See Past Women In Tech Summit</h2>
+        <div class="pt-1 underprogram"></div>
+      </div> 
+      <div class="past-img d-flex py-2">
+        <div class="virtual1 mx-2"> 
+          <div class="vir-cover">
+            <a href="https://womenintechsummit.net/spring-2022-virtual-summit/"><h2 class="py-5 my-4">SPRING 2022 <br>VIRTUAL SUMMIT</h2></a>
+          </div> 
+        </div>
+        <div class="virtual2 mx-2">
+          <div class="vir-cover">
+            <a href="https://womenintechsummit.net/past-events/spring2021virtual/"><h2 class="py-5 my-4">SPRING 2021 <br>VIRTUAL SUMMIT</h2></a>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Please correct the alignment of these two images on the mobile version. As you can see from the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/198262051-60c69214-a368-402b-be05-889545f414c0.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/198262145-15a8a686-9f7e-4ff1-b259-2027e790e145.png)

- [ ] Also please consider adding the orange color overlay on these two images 😉